### DISCUSSION TITLE
Math symbols in Table caption

### DISCUSSION THREAD

**Discussion by giorgiopaulon** [davidgohel/flextable#521] [Q&A] [answered]
I am interested in exporting a flextable to a .pdf document using Quarto / Markdown. The only resources I found suggest to use the following

```
#| tbl-cap: !expr sprintf("A caption with some math, $\\gamma \\geq 0.5$, ($N = %s$)", nrow(iris))
head(iris) %>% 
  flextable()
```

However, this results in the following

![Screenshot 2023-03-14 at 1 18 10 PM](https://user-images.githubusercontent.com/90702867/225100488-6b9dcb2f-b330-4b15-9a4b-2e43599fbcfd.png)

**Comment by davidgohel** [comment, marked as answer]
Quarto manage captions instead of the packages. We only emit an empty caption https://github.com/davidgohel/flextable/blob/master/R/captions.R#L252

I am afraid we can't help.

**Reply by giorgiopaulon** [reply]
It is indeed a Quarto bug.
